### DISCUSSION TITLE
How to handle multi-sample multi-ome data in SCENIC+?

### DISCUSSION THREAD

**Discussion by SidG13** [aertslab/scenicplus#153] [Q&A]
Hi all,

I have a multi-sample multiome dataset that I integrated in Seurat using Harmony to remove a batch effect between my samples. I'm not sure if SCENIC+ can currently handle this sort of data but I'm curious if there are paths forward here. In both my RNA and ATAC data, cells have been differentiated between the samples by an added flag (ie., "A_", "B_", "C_") to the start of the cell barcode.

Here's a description of my basic inputs to SCENIC+:
**RNA**: A raw counts matrix (i.e., derived from in Seurat: MergedObject@assays$RNA@counts), where cells from different samples have added flags differentiating them. 
**ATAC**: A concatenated, sorted fragments files between my samples. 

Of course these inputs alone don't account for the batch effect, but I'm wondering if there was some way to differentiate samples based on flags in the cell barcode? Is there any way to incorporate multiple samples at this time, or is the only option to run each sample separately and somehow merge results at the end?

Thanks!

**Comment by SeppeDeWinter** [comment]
Hi @SidG13 

You can for sure use all samples at the same time, this is no issue at all.

To get you started I suggest to have a look at https://pycistopic.readthedocs.io/en/latest/Cortex_pycisTopic.html.

Best,

Seppe

**Reply by maithermbarros** [reply]
Hi @SeppeDeWinter. I am also trying to use multiome datasets that have been integrated into a single object, but the link you posted above does not work. Can you please share another tutorial? Thank you!

**Reply by SeppeDeWinter** [reply]
Hi @maithermbarros 

This notebook can still be found here: https://github.com/aertslab/pycisTopic/blob/old/notebooks/Cortex_pycisTopic.ipynb

All the best,

Seppe

**Comment by SidG13** [comment]
Thanks for the response! 

As a follow-up, in your opinion, is there a reason to not run each sample separately through the SCENIC+ pipeline and merge results at the end (since normalization/cell selection is being done on each sample independently)? I realize it may take a little longer. My plan was to do this and just merge the eRegulon predictions at the end. Some samples have many more cells than others, so it may not do a good job predicting the eRegulons for some genes, but other samples may. 

Separately, I'm just wondering what steps do I need to do from the vignette you linked to, and when do I switch back to using the regular SCENIC+ pipeline? 

Thanks!
Sid

**Comment by SeppeDeWinter** [comment]
Hi @SidG13 

I would suggest running all samples at the same time.
Merging results afterwards is quite hard and the more cells you have the better the inference will be.

Best,

Seppe
